pkg/ddevapp/global_dotddev_assets/commands/host/phpmyadmin

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Prompt downloading phpMyAdmin if it is not installed.
+## Usage: phpmyadmin
+## Example: "ddev phpmyadmin"
+
+if [ "${DDEV_NONINTERACTIVE:-}" != "" ]; then
+  echo "Nothing has been changed."
+  exit 0
+fi
```

Comment:
Let's change the indentation here to 4 spaces, as in the entire script. And I will test it again.

```suggestion
if [ "${DDEV_NONINTERACTIVE:-}" != "" ]; then
    echo "Nothing has been changed."
    exit 0
fi
```